db/migrate/20181026182144_add_tasks_table.rb

```diff
@@ -0,0 +1,12 @@
+class AddTasksTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tasks, :id => :bigserial do |t|
```

Comment:
We probably also need columns for `resource_type`, `resource_id`, `method` and `arguments` right?

Reply:
`resource` so that we can link the task to what was created?  I don't want to limit it to that necessarily since there could be tasks that don't relate to any object or tasks which create multiple objects.  Maybe a jsonb context column ?

Reply:
No, how does the task know what method to call on which instance / class?

Reply:
Oh I see, I wasn't thinking that we would have the task actually tell the operations worker what to do, it is primarily for tracking.  We might need an MiqQueue equivalent to do what you're talking about.